packages/@okta/vuepress-site/docs/guides/delete-all-stay-signed-in-sessions/main/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Delete all Stay signed in sessions
+---
+
+<ApiLifecycle access="ie" />
+
+This guide explains how to delete all sessions of the **Stay signed in** feature using the [My Account Sessions API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/Sessions/). See [Stay signed in](https://help.okta.com/okta_help.htm?type=oie&id=ext-stay-signed-in) for more details.
+
+> **Note:** This document is written for Okta Identity Engine. If you’re using Classic Engine, time to upgrade. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
+
+---
+
+**Learning outcomes**
+
+- Learn to enable the usePersistentCookie option.
+- Understand how to delete all sessions using the API.
+
+**What you need**
+
+- An Okta developer org configured for a **Stay signed in** use case
+- Access to the [My Account Sessions API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/Sessions/)
+- [Postman client](https://www.getpostman.com/downloads/) to set up `userPersistentCookie`. See [Get Started with the Okta APIs](https://developer.okta.com/code/rest/) for information on setting up Postman.
+
+**Sample code**
+
+[Request example](#request-example)
+
+---
+
+## About **Stay signed in**
+
+**Stay signed in** lets users establish an Okta session that continues across browsers and devices. When a user selects this option during authentication, they won't be prompted again for MFA for the amount of time defined in your global session policy.
```

Comment:
```suggestion
**Stay signed in** lets users establish an Okta session that continues after they close and reopen their browsers. When a user selects this option during authentication, they won't be prompted again for MFA for the amount of time defined in your global session policy.
```

Reply:
I had to change my opening line in the product docs for this reason, stressing that it didn't cross devices necessarily